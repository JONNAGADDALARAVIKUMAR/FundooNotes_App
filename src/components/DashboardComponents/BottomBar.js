import React, { Component } from 'react';
import {Appbar} from 'react-native-paper';
import DashBoardScreenStyles from '../../styles/DashBoardScreenStyles';

export default class ToolBar extends Component {
    render() {
        return(
            <Appbar style = {DashBoardScreenStyles.App_Bar_Style_Bottom}>
                <Appbar.Action
                    icon="check-box-outline"
                    style = {{margin: '5%'}}
                    onPress = {() => console.log('Clicked Square checkBox')}
                />
                <Appbar.Action
                    style = {{margin: '5%'}}
                    icon="brush"
                    onPress = {() => console.log('Clicked Brush')}
                />
                <Appbar.Action
                    style = {{margin: '5%'}}
                    icon="microphone-outline"
                    onPress = {() => console.log('Clicked Micro phone')}
                />
                <Appbar.Action
                    style = {{margin: '5%'}}
                    icon = "panorama"
                    onPress={() => console.log('Clicked panorama')}
                />
                <Appbar.Content/>
                <Appbar.Action
                    style = {DashBoardScreenStyles.Add_Notes_Icon_Style}
                    icon = "plus"
                    onPress={() => this.props.navigation.navigate('AddNewNotes')}
                />
            </Appbar>
        )
    }
}