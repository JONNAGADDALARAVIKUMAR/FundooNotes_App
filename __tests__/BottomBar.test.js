import React from 'react';
import {shallow} from 'enzyme';
import BottomBar from '../src/components/DashboardComponents/BottomBar';
import {Appbar} from 'react-native-paper';

describe('test BottomBar Screen', () => {
    it('test when render should match to snapshot', async () => {
        const component = shallow(<BottomBar/>)
        expect(component).toMatchSnapshot();
    })
    it('test appbar action in bottom bar component should have length 5', () => {
        const component = shallow(<BottomBar />)
        expect(component.find(Appbar.Action)).toHaveLength(5)
        expect(component.find(Appbar.Action).at(0).props().icon).toEqual('check-box-outline')
        expect(component.find(Appbar.Action).at(1).props().icon).toEqual('brush')
        expect(component.find(Appbar.Action).at(2).props().icon).toEqual('microphone-outline')
        expect(component.find(Appbar.Action).at(3).props().icon).toEqual('panorama')
        expect(component.find(Appbar.Action).at(4).props().icon).toEqual('plus')
    })
})