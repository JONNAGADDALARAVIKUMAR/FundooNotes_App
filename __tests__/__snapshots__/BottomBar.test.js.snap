// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test BottomBar Screen test when render should match to snapshot 1`] = `ShallowWrapper {}`;
